example/lib/main.dart

```diff
@@ -20,6 +21,9 @@ class _MyAppState extends State<MyApp> {
   String _platformVersion = Strings.valueUnknown;
   bool _initializeLocatorPlugin = false;
   bool _checkPermissions = false;
+  var _latitude = Strings.noLatitude;
+  var _longitude = Strings.noLongitude;
+  Map<dynamic, dynamic> _coordinates = {};
```

Comment:
_coordinates can be deleted from this scope, and latitude/longitude will be inside the new class that I explain in the comment below

Reply:
done